src/glabs-inline/web/index.scss

```diff
@@ -0,0 +1,82 @@
+@import '_core';
+
+a {
+  text-decoration: none;
+}
+
+h1, p {
+  margin: 0;
+}
+
+.container {
+  font-family: "Guardian Text Sans Web","Helvetica Neue",Helvetica,Arial,"Lucida Grande",sans-serif;
+  background-color: $neutral-8;
+  border-top: 2px solid $paid-article-brand;
+
+  display: flex;
+  flex-direction: row;
+  max-width: gs-span(8);
+  height: 110px
+}
+
+.margin {
+  margin: 5px;
+}
+
+.body {
+  flex-grow: 1;
+  flex-shrink: 1;
+  flex-basis: 0;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.thumbnail {
+  max-width: 35%;
```

Comment:
Might be worth switching to:

```
max-width: 184px;
width: 35%; 
```

Firefox sometimes gets unhappy if max-width is used without the width being set.

The value of 184px is taken from the typical trail image dimension if the height is restricted to 110px. This allows the height of the carrot to be the deciding factor, but will remove the extra whitespace from Firefox:

<img width="634" alt="picture 909" src="https://user-images.githubusercontent.com/8607683/29078447-5c8da39c-7c52-11e7-8297-c5fc778ab9b6.png">

Thoughts?